docs/src/developer/DEV_CONTAINER.md

```diff
@@ -0,0 +1,86 @@
+# VSCode Dev Containers
+
+VSCode Dev Containers are a great way to develop as they can contain all of the project's dependencies and in the end you have a read-to-use dockerFile which can also be used in your CI/CD pipeline.
+
+It provides a normalized experience to all VSCode users across MacOS, Windows and Linux. All you need to get going is a the open source docker engine (not Docker Desktop!) and docker-compose.
+
+More information about Dev Containers can be found from the [official documentation](https://code.visualstudio.com/docs/devcontainers/containers).
+
+## Setting up Docker (OSS - Variant)
```

Comment:
You could just reference the other `INSTALLING_DOCKER.md` doc here.

Reply:
Yeah you're right...I thought I updated this (I hope I didn't force push over the changes, as I swear I did link to the `INSTALLAING_DOCKER.md` before...I will prepare another PR.